src/applications/claims-status/utils/helpers.js

```diff
@@ -1077,22 +1125,9 @@ export function setDocumentRequestPageTitle(displayName) {
 
 // Used to set page title for the CST Tabs
 export function setTabDocumentTitle(claim, tabName) {
-  if (claimAvailable(claim)) {
-    const claimDate = buildDateFormatter()(claim.attributes.claimDate);
-    const claimType = getClaimType(claim);
-    const title =
-      tabName === 'Files'
-        ? `${tabName} For ${claimDate} ${claimType} Claim`
-        : `${tabName} Of ${claimDate} ${claimType} Claim`;
-    setDocumentTitle(title);
-  } else {
-    const title =
-      tabName === 'Files'
-        ? `${tabName} For Your Claim`
-        : `${tabName} Of Your Claim`;
-    setDocumentTitle(title);
-  }
+  setDocumentTitle(generateClaimTitle(claim, 'document', tabName));
```

Comment:
This could probably be further simplified, but I wanted to reduce the amount of changes to the existing code.